frontend/cypress/integration/components/communityLinks.spec.js

```diff
@@ -0,0 +1,44 @@
+describe('Community Links Landing Page', () => {
+    beforeEach(() => {
+      cy.visit('/');
+    });
+  
+    it('finds whether the text and images are displayed', () => {
+      // TODO: fix padding according to style scape and add to testing
+
+      // Making sure that the community links are included in the landing page
+      cy.get('[data-cy=communitLinks]');
+      // making sure that the join us text is there
+      cy.get('[data-cy=communityLinks-title]');
+      // making sure that the body text is there
+      cy.get('[data-cy=communityLinks-bodyText]');
+      // making sure that the body text is there
+      cy.get('[data-cy=communityLinks-joinCommunityText]');
+      // making sure that the body text is there
+      cy.get('[data-cy=communityLinks-images]');
+
+      // making sure that each image is being displayed, and is visible after hovering
+      // TODO: figure out how to test the zoom feature, and also add sizing into it aswell
+      cy.get('[data-cy=communityLinks-facebookImage].zoom')
+        .trigger('mouseover')
+      cy.get('[data-cy=communityLinks-slackImage].zoom')
+        .trigger('mouseover')
+      cy.get('[data-cy=communityLinks-discordImage].zoom')
+        .trigger('mouseover')
+    });
+
```

Comment:
If a ticket can be created for the `TODO` but not put in the upcoming sprint.

Reply:
There's an easier way to do this I believe but don't have the time to look into it until after milestone.